src/Eloquent/Builder.php

```diff
@@ -30,24 +30,24 @@ class Builder extends EloquentBuilder
         'avg',
         'count',
         'dd',
-        'doesntExist',
+        'doesntexist',
         'dump',
         'exists',
-        'getBindings',
-        'getConnection',
-        'getGrammar',
+        'getbindings',
+        'getconnection',
+        'getgrammar',
         'insert',
-        'insertGetId',
-        'insertOrIgnore',
-        'insertUsing',
+        'insertgetid',
+        'insertorignore',
+        'insertusing',
         'max',
         'min',
         'pluck',
         'pull',
         'push',
         'raw',
         'sum',
-        'toSql',
+        'tosql',
```

Comment:
This should be removed. `SQL` doesn't make sense here.

Reply:
Do you mean the whole entry (`toSql`) or to leave it camel cased?

Reply:
```php
User::query()->toSql();
```
Gives the following error
> BadMethodCallException: This method is not supported by MongoDB. Try "toMql()" instead.

So I guess it should be removed completely. Maybe replace it with the mentioned method, but that's your call.

Reply:
Yes, remove the method `tosql` from the list.
Adding `tomql` would be helpful.